schemas/telemetry/frecency-update/frecency-update.1.schema.parquetmr.txt

```diff
@@ -7,6 +7,7 @@ message frecency-update {
     required int32 num_chars_typed;
     required int32 rank_selected;
     required int32 num_suggestions_displayed;
+    required binary study_variation;
```

Comment:
This field should be added as `optional`, since there will be historic data where it is missing.

Reply:
See the other comment regarding this.